common/crypto.go

```diff
@@ -24,6 +27,10 @@ const (
 	cost = 13
 )
 
+func init() {
```

Comment:
This package was calling crypto-related stuff without a guaranteed-to-be-seeded PRNG